backend/apps/owasp/models/event.py

```diff
@@ -3,56 +3,46 @@
 from django.db import models
 
 from apps.common.models import BulkSaveModel, TimestampedModel
-from apps.owasp.models.common import RepositoryBasedEntityModel
 
 
-class Event(BulkSaveModel, RepositoryBasedEntityModel, TimestampedModel):
+class EventCategory(models.TextChoices):
+    """Event category choices."""
+
+    GLOBAL = "global", "Global"
+    APPSEC_DAYS = "appsec_days", "AppSec Days"
+    PARTNER = "partner", "Partner"
+    OTHER = "other", "Other"
+
+
+class Event(BulkSaveModel, TimestampedModel):
     """Event model."""
 
     class Meta:
         db_table = "owasp_events"
         verbose_name_plural = "Events"
 
-    level = models.CharField(verbose_name="Level", max_length=5, default="", blank=True)
-    url = models.URLField(verbose_name="URL", default="", blank=True)
+    category = models.CharField(
+        verbose_name="Category",
+        max_length=20,
+        choices=EventCategory.choices,
+        default=EventCategory.OTHER,
+    )
 
-    owasp_repository = models.ForeignKey(
-        "github.Repository", on_delete=models.SET_NULL, blank=True, null=True
+    category_description = models.TextField(
```

Comment:
You can remove this field. The description should belong to category entity, not to the event as it's always tied to the category.